packages/sit-onyx/src/styles/variables-dark.css

```diff
@@ -0,0 +1,429 @@
+/**
+ * Do not edit directly.
+ * This file contains the specific variables for the "dark" theme.
+ * Imported from Figma API on Tue, 09 Jan 2024 08:17:34 GMT
+ */
+:root {
```

Comment:
maybe instead of using the cli with `-mode onyx light dark` we split it up again and add another parameter which can overwrite :root?
yeah then we need to run the cli three times like
`-mode onyx`
`-mode dark -selector html.dark`
`-mode light -selector html.light`
but it might help this situation?

Reply:
also maybe instead of using html.dark we can use a class like 
.onyx--light and .onyx--dark

because probably the apps will want to have an implementation which saves the last choice and does not just use the system settings, and it might not be directly set on the hmtl tag but somewhere else

Reply:
> maybe instead of using the cli with `-mode onyx light dark` we split it up again and add another parameter which can overwrite :root?


Yeah thats probably how we have to do it...another would be to hard code a check if the theme is "dark" or "light" and set the selector accordingly but I think that is not really transparent to the user

Reply:
I added and changed the selector in #96